airflow/executors/executor_loader.py

```diff
@@ -77,14 +77,36 @@ def load_executor(cls, executor_name: str) -> BaseExecutor:
         """
         if executor_name == CELERY_KUBERNETES_EXECUTOR:
             return cls.__load_celery_kubernetes_executor()
+        try:
+            executor_cls, import_source = cls.import_executor_cls(executor_name)
+            log.debug(
+                "Loading executor %s from %s", executor_name, import_source.replace("custom", "custom path")
+            )
+        except ImportError as e:
+            log.error(e)
+            raise AirflowConfigException(
+                f'The module/attribute could not be loaded. Please check "executor" key in "core" section. '
```

Comment:
```suggestion
                f'The custom executor could not be loaded. Please check "executor" key in "core" section. '
```
WDYT? I think it would be more detailed